docs/cookbook.rst

```diff
@@ -125,4 +125,55 @@ On the subscriber's customer object, use the charge method to generate a Stripe
     amount = Decimal(10.00)
     customer.charge(amount)
 
-Source code for the Customer.charge method is at https://github.com/pydanny/dj-stripe/blob/master/djstripe/models.py#L573-L596
\ No newline at end of file
+Source code for the Customer.charge method is at https://github.com/pydanny/dj-stripe/blob/master/djstripe/models.py#L573-L596
+
+REST api
+--------
+
+The subscriptions can be accessed through a REST api. Make sure you have
```

Comment:
thanks for checking this, it's well understandable in both cases, my comment was more about using "API" rather than "api"